pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "todo-app"
+version = "0.1.0"
+description = "A simple ToDo app for your DevOps apprenticeship"
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.7"
+flask = "^1.1.2"
+python-dotenv = "^0.14.0"
+requests = "^2.25.1"
+pendulum = "^2.1.2"
+selenium = "^3.141.0"
```

Comment:
selenium should also be in the dev-dependencies section

Reply:
Added selenium = "^3.141.0" to dev dependencies